esp-metadata-generated/src/_generated_esp32.rs

```diff
@@ -1518,3 +1518,284 @@ macro_rules! define_io_mux_reg {
         }
     };
 }
+#[macro_export]
+#[doc(hidden)]
+macro_rules! implement_alternate_function_markers {
+    () => {
+        #[instability::unstable]
+        #[doc = "Marker trait for pins that have the SD_CMD input function"]
+        pub trait InputFunction_SD_CMD {}
+        #[instability::unstable]
+        #[doc = "Marker trait for pins that have the SD_DATA0 input function"]
+        pub trait InputFunction_SD_DATA0 {}
```

Comment:
I'm pretty OK with the naming style here, but I didn't place the clippy allow just so that we have some noise about them, in case we want to go with CamelCase.